spec/lib/topics_filter_spec.rb

```diff
@@ -1821,4 +1909,62 @@
       end
     end
   end
+
+  describe "custom filter mappings for in: and status: operators" do
+    fab!(:topic)
+    fab!(:solved_topic) { Fabricate(:topic, closed: true) }
+
+    describe "custom in: filter" do
+      before do
+        plugin_instance = Plugin::Instance.new
```

Comment:
you are initializing this here but then doing `Plugin::Instance.new.add_filter_custom_filter` below, is it needed ?